runbotics/runbotics-orchestrator-ui/src/views/process/ProcessRunView/SavedSchedule.tsx

```diff
@@ -74,14 +75,15 @@ const SavedSchedule: FC<SavedScheduleProps> = ({ processId }) => {
         await dispatch(scheduleProcessActions.getSchedulesByProcess({ processId }));
     };
 
-    const humanReadableCron = (cronExpression: string) => translate(
+    const humanReadableCron = (cronExpression: string) => (
+            translate(
         'Process.Schedule.Cron.HumanReadable',
         {
             cron: cronstrue
-                .toString(cronExpression)
-                .replace('At', translate('Process.Schedule.Cron.At')),
+                .toString(cronExpression, {locale: moment.locale()}).toLowerCase()
```

Comment:
so this lib provides messages in different languages? Make sure that it is correct + add i18n locale, do not base on moment

Reply:
Yes, it's correct. Without it expressions at Process -> RUN -> schedule process are not being translated.

**Without** {locale:moment.locale()}:

![image](https://user-images.githubusercontent.com/69820069/191293745-71da6895-88ac-4ef8-904a-7a330829d874.png)

**With** {locale: moment.locale()}:
![image](https://user-images.githubusercontent.com/69820069/191294129-8967ac64-6efc-4648-9e8c-dbbfba56a6e9.png)


replaced moment with i18n ✔